frontend/vue/components/constants/megaMenuLinks.ts

```diff
@@ -11,8 +12,11 @@ const sectionPulses = 'pulses'
 const sectionApps = 'apps'
 const sectionCircuits = 'circuits'
 const sectionLabs = 'labs'
+const sectionTutorials = 'tutorials'
 
-const baseUrl = window && window.location ? window.location.origin : 'https://learn.qiskit.org'
+const baseUrl = 'https://learn.qiskit.org'
```

Comment:
Everything looks great here, @korgan00 

I only have 1 question:
Since the Textbook is already on `https://learn.qiskit.org` — in this instance, do we need the `baseUrl`?

In other words, should we be using relative paths here in Platypus? 🤔

Reply:
Yeah, sounds good. I will take a look at it